vispy/gloo/buffer.py

```diff
@@ -449,6 +453,9 @@ def _prepare_data(self, data, convert=False):
             self._last_dim = c
         return data
 
+    def set_instancing_divisor(self, divisor):
```

Comment:
Does this need to be a public method? If not, does it need to be a method at all?

Reply:
I figured it would be nice to expose a way to change this afterwards, but It's not strictly needed.

Reply:
Yeah, good point!